### ISSUE TITLE
question about cls_modules

### ISSUE THREAD

**Issue by freshman2001** [Turoad/CLRNet#34]
Hi! Thank you so much for this valuable work for the community. I would like to ask, in line 73 of clrnet/model/head/clrhead.py, is the role of self.cls_modules to judge whether the priority contains lane line samples?

**Comment by MySuperSoul** [comment]
> Hi! Thank you so much for this valuable work for the community. I would like to ask, in line 73 of clrnet/model/head/clrhead.py, is the role of self.cls_modules to judge whether the priority contains lane line samples?

Not exactly, our head consists of two branchs, one branch for classification and another for regression. The `cls_modules` is used to generate classification-related feature from original extracted lane feature, which consists of three `LinearModule`.  After `cls_modules` there will be one linear layer`self.cls_layers = nn.Linear(self.fc_hidden_dim, 2)` to output cls result.
